system/db/skills.txt

```diff
@@ -200,7 +200,7 @@
 { skillId: 10206, className: "Highlander_ScullSwing", name: "Skull Swing", useType: "MeleeGround", attackType: "Strike", attribute: "Melee", classType: "Melee", maxLevel: 5, enableAngle: 90, maxRange: 100, waveLength: 40, splashType: "Square", splashRange: 13, splashHeight: 0, splashAngle: 0, splashRate: 1, factor: 100, factorByLevel: 0, atkAdd: 0, atkAddByLevel: 0, defaultHitDelay: 0, deadHitDelay: 50, shootTime: 600, delayTime: 0, cancelTime: 600, hitTime: [200], holdTime: [400], speedRate: 1, speedRateAffectedByDex: false, speedRateAffectedByBuff: false, enableCastMove: false, knockDownType: "KnockBack", knockDownVelocity: 220, knockDownVAngle: 10, reinforceAbility: "Highlander25", cooldownGroup: "ScullSwing", cooldownTime: 40000 },
-{ skillId: 10207, className: "Highlander_SkyLiner", name: "Skyliner", useType: "MeleeGround", attackType: "Slash", attribute: "Melee", classType: "Melee", maxLevel: 1, basicSp: 54, enableAngle: 90, maxRange: 100, waveLength: 40, splashType: "Square", splashRange: 13, splashHeight: 0, splashAngle: 0, splashRate: 0, factor: 297.2, factorByLevel: 49.5, atkAdd: 0, atkAddByLevel: 0, defaultHitDelay: 0, deadHitDelay: 50, shootTime: 9999, delayTime: 0, cancelTime: 9999, hitTime: [200], holdTime: [400], speedRate: 1, speedRateAffectedByDex: true, speedRateAffectedByBuff: false, enableCastMove: false, reinforceAbility: "Highlander29", hiddenReinforceAbility: "Highlander40", hiddenReinforceAbilityFactorByLevel: "1.25", cooldownGroup: "SkyLiner", cooldownTime: 15000, overheatGroup: "SkyLiner", overheatCount: "0", overheatDelay: "15000" },
+{ skillId: 10207, className: "Highlander_SkyLiner", name: "Skyliner", useType: "MeleeGround", attackType: "Slash", attribute: "Melee", classType: "Melee", maxLevel: 1, basicSp: 54, enableAngle: 90, maxRange: 100, waveLength: 40, splashType: "Square", splashRange: 13, splashHeight: 0, splashAngle: 0, splashRate: 0, factor: 297.2, factorByLevel: 49.5, atkAdd: 0, atkAddByLevel: 0, defaultHitDelay: 0, deadHitDelay: 50, shootTime: 9999, delayTime: 0, cancelTime: 9999, hitTime: [200], holdTime: [400], speedRate: 1, speedRateAffectedByDex: true, speedRateAffectedByBuff: false, enableCastMove: false, reinforceAbility: "Highlander29", hiddenReinforceAbility: "Highlander40", hiddenReinforceAbilityFactorByLevel: "1.25", cooldownGroup: "SkyLiner", cooldownTime: 15000, overheatGroup: "SkyLiner_OH", overheatCount: "0", overheatDelay: "15000" },
```

Comment:
Have you confirmed that this is actually the overheat group sent for this skill via `ZC_OVERHEAT_CHANGED`? While it does seem like it probably should be `SkyLiner_OH`, the official skill data says it's `SkyLiner`.

Reply:
The only confirmation I can give is that if it is set to just Skyliner, the skill doesn't go on cooldown when it is used, but with Skyliner_OH it does.